blueprints/app/files/package.json

```diff
@@ -13,11 +13,12 @@
   "scripts": {
     "build": "ember build --environment=production",
     "lint": "npm-run-all --print-name --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
-    "lint:fix": "npm-run-all --print-name --aggregate-output --continue-on-error --parallel \"lint:*:fix\"",
+    "lint:fix": "npm-run-all \"lint:fix:script -- .\"",
+    "lint:fix:script": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:*:fix -- {@}\" --",
     "lint:hbs": "ember-template-lint .",
-    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:hbs:fix": "ember-template-lint --fix",
     "lint:js": "eslint . --cache",
-    "lint:js:fix": "eslint . --fix",
+    "lint:js:fix": "eslint --fix",
```

Comment:
It seems removing the `.` makes the script unusable via the terminal. It doesn't fix lint violations anymore.

Reply:
would solving this by making a `:script` version be the preferred way?